tests/data/samplers/max_tokens_batch_sampler_test.py

```diff
@@ -64,34 +58,27 @@ def test_guess_sorting_key_picks_the_longest_key(self):
         assert sampler.sorting_keys == ["passage"]
 
     def test_from_params(self):
```

Comment:
We don't need these tests anymore, because no classes actually implement `from_params` themselves.